_data-prepper/common-use-cases/s3-logs.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: S3 logs
+parent: Common use cases
+nav_order: 20
+---
+
+# S3 logs
+
+Data Prepper allows you to load logs from [Amazon Simple Storage Service](https://aws.amazon.com/s3/) (Amazon S3), including traditional logs, JSON documents, and CSV logs.
+
+
+## Architecture
+
+Data Prepper can read objects from S3 buckets using an [Amazon Simple Queue Service (SQS)](https://aws.amazon.com/sqs/) (Amazon SQS) queue and [Amazon S3 Event Notifications](https://docs.aws.amazon.com/AmazonS3/latest/userguide/NotificationHowTo.html).
+
+Data Prepper polls the Amazon SQS queue for S3 event notifications. When Data Prepper receives a notification that an S3 object was created, Data Prepper reads and parses that S3 object.
+
+The following diagram shows the overall architecture of the components involved.
+
+<img src="{{site.url}}{{site.baseurl}}/images/data-prepper/s3-source/s3-architecture.jpg" alt="S3 source architecture">{: .img-fluid}
+
+The flow of data is as follows.
+
+1. A system produces logs into the S3 bucket.
```

Comment:
I'm not sure one can "produce into". Do we mean "A system sends logs to an S3 bucket"?